bee-message/src/payload/cpt2/indexation/padded.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// An indexation payload index padded with `0` up to the maximum length.
+#[derive(Clone, Copy, Eq, PartialEq, Hash)]
+#[cfg_attr(doc_cfg, doc(cfg(feature = "cpt2")))]
+pub struct PaddedIndex([u8; PaddedIndex::LENGTH]);
```

Comment:
Do you actually make use of this ? It existed in bee for KV storage purposes only.

Reply:
Nope, we don't